logs/client.go

```diff
@@ -196,10 +196,6 @@ func (c *Client) Run(ctx context.Context) error {
 			return err
 		}
 
-		if logs.NextPage == "" {
-			break
-		}
-
 		if c.opts.follow && len(logs.Logs) == 0 {
 			time.Sleep(2 * time.Second)
 			continue
```

Comment:
This time.Sleep could also be moved to the end of the loop. A live tail should always have a NextPage though, so it doesn't actually matter.

Reply:
```go
		logs, err := c.getLogs(ctx, nextPage)
		if err != nil {
			return err
		}

		err = c.printResult(logs.Logs)
		if err != nil {
			return fmt.Errorf("failed to print result: %w", err)
		}

		if c.opts.follow && len(logs.Logs) == 0 {
			time.Sleep(2 * time.Second)
		}

		if logs.NextPage == "" {
			break
		}

		nextPage = logs.NextPage
```

Guards against the possibility of a missing NextPage (for some reason). Reasonable?

Reply:
I like it. 👍